examples/view_generator/terraform.tfvars

```diff
@@ -0,0 +1,27 @@
+project_id        = "example-project"
+time_partitioning = "DAY"
+dataset_labels = {
```

Comment:
Instead of making these into complex variables, can we just hard-code them into the example config?

Users should *not* need to complete a complex `terraform.tfvars` file to use an example.

Reply:
Can you address this? For examples specifically, users should *not* need a complex tfvars file. Ideally just things which are truly specific to them (ex. project-id).

Reply:
@milesmatthias do you have an update here?

Reply:
No, I no longer work on this project. This PR was passing all tests when I finished, but if someone would like to make other tweaks to this PR before merging, go for it.